tests/ui_tests/test_configs.py

```diff
@@ -360,7 +360,11 @@ def test_configs_fields(sdk_client_fs: ADCMClient, config_dict, app_fs):
     fields = ui_config.get_app_fields()
     with allure.step('Check save button status'):
         save_err_mess = f"Correct status for save button {[expected['save']]}"
-        assert expected['save'] == ui_config.save_button_status(), save_err_mess
+        button_state = ui_config.save_button_status()
+        assert expected['save'] == button_state, save_err_mess
```

Comment:
Assertion message

Reply:
Take a closer look at `save_err_mess`

Reply:
I've deleted this message :D Yeah, saw it right after commented